src/registryctl/main.go

```diff
@@ -37,25 +38,15 @@ func (s *RegistryCtl) Start() {
 		Handler: s.Handler,
 	}
 
-	if s.ServerConf.Protocol == "HTTPS" {
-		tlsCfg := &tls.Config{
-			MinVersion:               tls.VersionTLS12,
-			CurvePreferences:         []tls.CurveID{tls.CurveP521, tls.CurveP384, tls.CurveP256},
-			PreferServerCipherSuites: true,
-			CipherSuites: []uint16{
-				tls.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
-				tls.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,
-				tls.TLS_RSA_WITH_AES_256_GCM_SHA384,
-				tls.TLS_RSA_WITH_AES_256_CBC_SHA,
-			},
+	if s.ServerConf.Protocol == "https" {
```

Comment:
I think it's good practice to add supported cipher suite, why remove it?

Reply:
remove it to make it more general. Because if you communicate with another component like Nginx or other third-party services, the ciphers you both specify are different will cause a mismatch.
I think the best practice is to open a new PR to upgrade supported TLS on our components to version 1.3. It's faster and more secure.